tools/src/file_format/merger.rs

```diff
@@ -25,7 +25,7 @@ pub struct HashedStructured {
 /// merge the records and write them to the provided writer.
 ///
 /// This logic was extracted out from `merge-analyses.rs` for the purpose of
-/// being able to test its logic through the introduction of `cmd_merge_ananlyses`.
+/// being able to test its logic through the introduction of `cmd_merge_analyses`.
```

Comment:
?

Reply:
This may have been a problem with my (no longer used) keyboard.io model 1's that experience some level of chatter due to bad lubrication (https://community.keyboard.io/t/guide-a-second-step-to-fixing-key-chatter/4101/5) although it could also have been muscle memory going wrong!